x/staking/types/validator.go

```diff
@@ -349,6 +338,7 @@ func (v Validator) AddTokensFromDel(pool Pool, amount sdk.Int) (Validator, Pool,
 		panic("zero exRate should not happen")
 	}
 
+	// TODO: This is unobvious from the name of the function, either justify or move outside.
```

Comment:
I think this is actually obvious, we could increase the comment... delegator tokens are non-bonded, thus the ratio in the pool needs to be updated

Reply:
Well, it depends where the tokens *came* from - what if they were minted out of thin air, for example?

Reply:
I sustain my view, delegations do not directly receive minted tokens.  - we could add that as a contract in the function comment. or as a property of tokens in delegations.

Not sure the "thin air" example is relevant - minted tokens are always immediately included in the pool

Reply:
Hmm I'm not totally satisfied, but added as a function contract for now.